.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Please provide a clear and concise description of the Issue:**
+
+**Environment (please complete the following information):**
+ - OS: <!-- E.g. macOS, Ubuntu 19.04, etc -->
+ - Nextflow version: 
+
```

Comment:
We can ask the project specific information using 

```
nextflow info nextflow-hub/xyz 
```

Reply:
A quick question-
when the issue will be raised for a particular project repo like fastqc, spotyping etc. we will already know which project it belongs to. Do we still have to ask this info?
```nextflow
(base) root@tuffy:~/shruti/ncbi_to_dropbox# nextflow info nextflow-hub/fastqc
 project name: nextflow-hub/fastqc
 repository  : https://github.com/nextflow-hub/fastqc
 local path  : /root/.nextflow/assets/nextflow-hub/fastqc
 main script : main.nf
 revisions   : 
 * master (default)
   develop
   sharma-shruti-dev
   sharma-shruti-patch-1
(base) root@tuffy:~/shruti/ncbi_to_dropbox#
```

Reply:
@sharma-shruti , yeah that's need specifically for the `revision` information.

Reply:
ok adding this information